.github/workflows/ci.yml

```
@@ -70,7 +73,7 @@ jobs:
         ccache -s
 
     - name: Running small tests
-      if: github.event_name == 'pull_request'
+      if: github.event_name == 'pull_request' || github.even_name != 'schedule'
```

`event_name`?

---

:+1: How did i manage to get rid of the "t" when copy pasting?? :monkey:

```suggestion
      if: github.event_name == 'pull_request' || github.event_name != 'schedule'
```